react/src/components/atoms/forms/ErrorMessage/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SvgInputError from '../../icons/SvgInputError';
+import SvgInputSuccess from '../../icons/SvgInputSuccess';
+import './style.css';
+
+const ErrorMessage = ({inputId, message, status}) => {
```

Comment:
Should the inputId actually be the id attribute for the div being rendered?

Reply:
no that should be the id for <input> that the error message is tied to